src/components/TableObject/TableObject.stories.tsx

```diff
@@ -0,0 +1,230 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+
+import { TableObject, Props } from './TableObject';
+import Button from '../Button';
+import Heading from '../Heading';
+import Icon from '../Icon';
+import Table from '../Table';
+import TableBody from '../TableBody';
+import TableCell from '../TableCell';
+import TableHeader from '../TableHeader';
+import TableHeaderCell from '../TableHeaderCell';
+import TableObjectBody from '../TableObjectBody';
+import TableObjectHeader from '../TableObjectHeader';
+import TableRow from '../TableRow';
+import Toolbar from '../Toolbar';
+import ToolbarItem from '../ToolbarItem';
+
+export default {
+  title: 'Organisms/Tables/Table Object',
+  component: TableObject,
+  subcomponents: { TableObjectBody, TableObjectHeader },
+  parameters: {
+    axe: {
+      // TODO: re-enable when component is worked on
+      skip: true,
+    },
+    badges: [BADGE.BETA],
+  },
+} as Meta;
+
+const Template: Story<Props> = (args) => (
+  <TableObject>
```

Comment:
Make sure onclick handlers are working on the button once that is in place. I think we want to keep some sort of pattern. Go from default -> ascending -> descending -> default when clicked. That establishes the pattern, and then the data would just be manipulated to take that form.

Reply:
I added this logic, but right now the click handlers only affect the state of an individual `TableHeaderCell`. Downstream devs will need to implement all of the contextual logic, including resetting the sort statuses of all other `TableHeaderCell`s when a sort order is set on one of them. I didn't implement that here because it adds the complexity of wrapping components in a context and passing and manipulating state, etc.

Reply:
This is exactly the right thinking @csnizik .